### PR TITLE
add minion-certificate-manager.proto

### PR DESCRIPTION
**Pull request by BenjaminJ**
Add minion-certificate-manager proto

HS-1309

### CODE REVIEW

File: minion-certificate-manager/build.sh
Comment (geraldhumphries) [inline minion-certificate-manager/build.sh:1]: This is the same as the file in the flows-contracts folder. Maybe just move that one to the root of the repo?

File: minion-certificate-manager/pom.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
```
Comment (geraldhumphries) [inline minion-certificate-manager/pom.xml:43]: We can probably pull most of this pom file up into the root `pom.xml` (same with the one in the flows-contracts folder).

**Review by geraldhumphries — APPROVED** [review]
LGTM. I wouldn't put too much effort into my comments, BTO-230 still needs to be complete in order to hook into the pom files/script.

**Comment by kbakanov** [comment]
@BenjaminJ where or how will the artifacts be available? E.g. generated .go files.

**Comment by BenjaminJ** [comment]
@kbakanov it will be available through the BFF (rest-server), it's in this PR https://github.com/OpenNMS-Cloud/horizon-stream/pull/996
Check `getMinionCertificate`

File: minion-certificate-manager/build.sh
Comment (freddy61025) [reply minion-certificate-manager/build.sh:1]: create a script folder to hold the real one. And each sub-module using a wrapper script to call it.

**Review by geraldhumphries — APPROVED** [review]

**Pull request merged**